concepts/modularization/modularization_overview.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Modularization in Kyma
+---
+
+With Kyma’s modular approach, you can install exactly the Kyma components you want, making the installation very lightweight and adjusted to your business needs. Kyma provides three components to support the modular approach: The Module Manager, the Lifecycle Manager, and the Runtime Watcher.
+
+## Kyma’s Modular Approach
+
+![](assets/modular_approach.svg)
+
+## 1. Kyma Module
+
+A Kyma module contains all information required to install and run the associated components in a Kyma runtime:
+
+* A renderable Kubernetes manifest that includes the Kubernetes resources of the module
+* A default configuration for module chart resource installation
+* Optionally, further information
+
+All these assets are bundled into a single container image using the OCI image specification.
+
+> **TIP:** Learn how to build a module(ADD_LINK).
+
+## 2. Module Manager
+
+The Module Manager installs, uninstalls, and manages the Kyma modules in a local (single cluster mode) or remote setup. For example, it remotely manages and synchronizes Kyma module manifest resources in the SAP Kyma Runtime.
```

Comment:
```suggestion
The Module Manager installs, uninstalls, and manages the Kyma modules in a local (single cluster mode) or remote setup (in which one cluster acts as control-plane). For example, it remotely manages and synchronizes Kyma module resources in the SAP Kyma Runtime while the Manifest is located in the Control Plane.
```

Reply:
We should standardize the spelling of "control plane" - capitalized or not? Is it a Kubernetes resource?

```suggestion
The Module Manager installs, uninstalls, and manages the Kyma modules in a local (single cluster mode) or remote setup (in which one cluster acts as control plane). For example, it remotely manages and synchronizes Kyma module resources in the SAP Kyma Runtime, while the manifest is located in the control plane.
```